src/DayColumn.js

```diff
@@ -219,23 +217,155 @@ let DaySlot = React.createClass({
     })
   },
 
-  _slotStyle(startSlot, endSlot, leftOffset){
+  _getStyledEvents() {
+    let { min, startAccessor, endAccessor, step } = this.props
+
+    let events = this.props.events.sort((a, b) => {
+      let startA = +get(a, startAccessor)
+      let startB = +get(b, startAccessor)
+
+      if (startA === startB) {
+        return +get(b, endAccessor) - +get(a, endAccessor)
+      }
+
+      return startA - startB
+    })
+
+    let getSlot = (event, accessor) => event && positionFromDate(
+      get(event, accessor), min, this._totalMin
+    )
+
+    let isSibling = (idx1, idx2) => {
+      let event1 = events[idx1]
+      let event2 = events[idx2]
+
+      if (!event1 || !event2) return false
+
+      let start1 = getSlot(event1, startAccessor)
+      let start2 = getSlot(event2, startAccessor)
+
+      return (Math.abs(start1 - start2) < 60)
+    }
+
+    let isChild = (parentIdx, childIdx) => {
+      if (isSibling(parentIdx, childIdx)) return false
+
+      let parentEnd = getSlot(events[parentIdx], endAccessor)
+      let childStart = getSlot(events[childIdx], startAccessor)
+
+      return parentEnd > childStart
+    }
+
+    let getSiblings = (idx) => {
```

Comment:
I suppose we could, but then we also have to pass startAccessor, endAccessor and events each time. Do you prefer that?